content/docs/introduction/faq.md

```diff
@@ -11,15 +11,24 @@ toc: full-width
 ### What is Prometheus?
 
 Prometheus is an open-source systems monitoring and alerting toolkit
-with an active ecosystem. See the [overview](/docs/introduction/overview/).
+with an active ecosystem.
+It is the only system directly supported by [Kubernetes](https://kubernetes.io/) and the de facto standard across the [cloud native ecosystem](https://landscape.cncf.io/).
+See the [overview](/docs/introduction/overview/).
 
 ### How does Prometheus compare against other monitoring systems?
 
 See the [comparison](/docs/introduction/comparison/) page.
 
 ### What dependencies does Prometheus have?
 
-The main Prometheus server runs standalone and has no external dependencies.
+The main Prometheus server runs standalone as a single monolithic binary and has no external dependencies.
+
+#### Is this cloud native?
+
+Yes.
+
+Cloud native is a flexible operating model, breaking up old service boundaries to allow for more flexible and scalable deployments.
```

Comment:
If we want to mention cloud native stuff in here, IMO the main thing making Prometheus cloud native is service discovery integration (to work well in dynamic cloud environments). Of course, also its dimensional data model and ability to store a lot of data.

Reply:
```suggestion
Cloud native is a flexible operating model, breaking up old service boundaries to allow for more flexible and scalable deployments.

Prometheus's [service discovery](https://prometheus.io/docs/prometheus/latest/configuration/configuration/) integrates with most tools and clouds. It's dimensional data model and scale into the tens of million of active series allows it to monitor large cloud-native deployments.
```

Reply:
Like so?

Reply:
Yup! But:

"It's" -> "Its"
"tens of million" -> "tens of millions"

Reply:
```suggestion
Cloud native is a flexible operating model, breaking up old service boundaries to allow for more flexible and scalable deployments.

Prometheus's [service discovery](https://prometheus.io/docs/prometheus/latest/configuration/configuration/) integrates with most tools and clouds. Its dimensional data model and scale into the tens of millions of active series allows it to monitor large cloud-native deployments.
```